src/braket/schema_common/schema_base.py

```diff
@@ -75,6 +76,10 @@ def parse_raw_schema(json_str: str) -> BraketSchemaBase:
         schema = BraketSchemaBase.parse_raw(json_str)
         module = BraketSchemaBase.import_schema_module(schema)
```

Comment:
This issue was a result of a testing miss. Could we ensure that we add in a test now, so that this does not happen in the future? Here, we missed validating that import_schema_module works for all schemas declared in the package. Could you devise a test which could do that?